docs/PAYLOAD.md

```diff
@@ -1513,7 +1513,7 @@ However if you want your `on('notification')` event handler called but no notifi
 		"moredata": "Do more stuff",
 		"content-available": 1
 	},
-  "notId": 2 // unique ID you generate
+    "notId": 2 // unique ID you generate
```

Comment:
This one doesn't seem to be 100% consistent with the surroundings. Am I seeing things?

Reply:
hmm I see what you mean with the content above it. In my editor and in the markup, all of the indentation lines up. When I redid the indent, git indicated some additional changes on these lines, even though it 'looked' the same. I added it to the commit. Maybe some weird thing with spaces vs. indents?

Reply:
![image](https://user-images.githubusercontent.com/616464/29726578-a3d03cd6-89a6-11e7-9fc2-2f6a7aa634a5.png)